testing/test_cacheprovider.py

```diff
@@ -56,7 +56,7 @@ def test_error():
         assert result.ret == 1
         result.stdout.fnmatch_lines([
             "*could not create cache path*",
-            "*1 warnings*",
+            "*2 warnings*",
```

Comment:
What is the new warning here?  Shouldn't it get matched also?

Reply:
1st warning is
```
could not create cache path /tmp/pytest-of-feuillemorte/pytest-38/test_cache_failure_warns0/.pytest_cache/v/cache/lastfailed
```
and second is for new-first. don't need to create another test for it